clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -1676,29 +1678,24 @@ void FunctionVariableConstraint::handle_params
     FVConstraint *Empty = fromEmpty ? From : To;
```

Comment:
You should add a line just after the above `for` loop that merges `getDeferredParams()` from both. That is, if you had two no-argument prototypes, they might each have deferred parameters, and these should be combined.

Reply:
(After line 1674/1676)